modules/ROOT/pages/am-installing.adoc

```diff
@@ -114,23 +114,28 @@ When installing the Anypoint Monitoring agent, note the following:
 
 == SOCKS5 Proxy Setting
 
-If you need to proxy your monitoring data as it leaves the data center, you can use the command line parameter `-p` when you install the Anypoint Monitoring agent.
+Anypoint Monitoring agent uses a protocol that requires a SOCKS5 proxy. 
```

Comment:
Can we keep "If you need to proxy your monitoring data as it leaves the data center, " because the SOCKS proxy is required when the customer needs to use a proxy.